### ISSUE TITLE
Admin password is always reset during install

### ISSUE THREAD

**Issue by okelet** [ansible/awx#3915]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
When running multiple times the installer, the admin password is always reset. If I have changed the password from the web, after running the installer, the password is reset to the default.

##### ENVIRONMENT
* AWX version: 4.0.0
* AWX install method: kubernetes (microk8s)
* Ansible version:  ansible 2.7.10
* Operating System: Ubuntu 18.04 x64
* Web Browser: Firefox 66

##### STEPS TO REPRODUCE

* Run the installer
* Change the admin password from the web UI
* Run the installer again (for an upgrade, testing, etc)

##### EXPECTED RESULTS

Keep the password changed in the web UI.

##### ACTUAL RESULTS

The password is reset with the one in the inventory.

**Comment by kylecribbs** [comment]
Is your postgres server persistent?

**Comment by okelet** [comment]
Yes, it's is a deployment in AWS with Aurora Postgres. The block, that should contain a `when`, to only set the password if the installation is new, or is forced to reset:

https://github.com/ansible/awx/blob/devel/installer/roles/kubernetes/tasks/main.yml#L188

**Comment by ryanpetrello** [comment]
@Spredzy @shanemcd any thoughts on this?

**Comment by shanemcd** [comment]
Did you set the password either as an extra var or directly in the inventory? If you overrode that variable and it's still reseting to the default value, that's not expected.

**Comment by okelet** [comment]
I think that it doesn't matter if you set the password via extra var or inventory, as the block that resets the password (https://github.com/ansible/awx/blob/devel/installer/roles/kubernetes/tasks/main.yml#L188) doesn't contain a when directive, so is always executed, no matter is the user already exists. Perhaps an additional variable to force password reset would be useful, so that task would only be executed if the new variable is set; for new installations, the block that creates the user would still be valid.

**Comment by wenottingham** [comment]
It shouldn't reset it to the default; just set it to whatever's in the inventory/var. So when running an upgrade, make sure it has the same password settings.

**Comment by bmalynovytch** [comment]
Related to https://github.com/ansible/awx/issues/231 and extremely dangerous IMHO when you remove the default `admin`  to enforce security.
